build.gradle

```diff
@@ -0,0 +1,248 @@
+	testimpl.extendsFrom testImplementation
+}
+
+def fromURL = { url, name ->
+	File file = new File("$buildDir/download/${name}.jar")
+	file.parentFile.mkdirs()
+	if (!file.exists()) {
+		new URL(url).withInputStream { downloadStream ->
+			file.withOutputStream { fileOut ->
+				fileOut << downloadStream
+			}
+		}
+	}
+	files(file.absolutePath)
+}
+
+dependencies {
+
+	// ### General application dependencies ###
+	
+	implementation fromURL(
+		'https://github.com/kbase/jars/raw/master/lib/jars/kbase/auth/kbase-auth-0.4.4.jar',
+		'kbase-auth-0.4.4'
+	)
+	implementation fromURL(
+		'https://github.com/kbase/jars/raw/master/lib/jars/kbase/kidl/kbase-kidl-parser-1409261812-7863aef.jar',
+		'kbase-kidl-parser-1409261812-7863aef'
+	)
+	implementation 'com.fasterxml.jackson.core:jackson-annotations:2.9.9'
+	implementation 'com.fasterxml.jackson.core:jackson-core:2.9.9'
+	implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.9'
+	implementation 'org.ini4j:ini4j:0.5.2'
+	implementation 'commons-io:commons-io:2.4'
+	implementation 'org.apache.commons:commons-lang3:3.1'
+	implementation 'commons-codec:commons-codec:1.8'
+	implementation 'info.picocli:picocli:4.6.1'
+	implementation 'org.mongodb:mongodb-driver-core:4.11.1'
+	implementation 'org.mongodb:mongodb-driver-sync:4.11.1'
+	implementation 'org.mongodb:bson-record-codec:4.11.1'
+	implementation 'org.mongodb:bson:4.11.1'
+	implementation 'org.slf4j:slf4j-api:1.7.30'
+	implementation 'ch.qos.logback:logback-classic:1.1.2'
+	implementation 'com.google.guava:guava:14.0.1'
+	implementation 'com.github.ben-manes.caffeine:caffeine:2.9.3'
+	implementation 'org.apache.kafka:kafka-clients:2.1.0'
+	implementation fromURL(
+		'https://github.com/kbase/jars/raw/master/lib/jars/kbase/handle/AbstractHandleClient-1.0.0.jar',
+		'AbstractHandleClient-1.0.0'
+	)
+	implementation fromURL(
+		'https://github.com/kbase/jars/raw/master/lib/jars/kbase/sample/SampleServiceClient-0.1.1.jar',
+		'SampleServiceClient-0.1.1'
+	)
+	
+	// ### Server dependencies, specifically for java_common JsonServerServlet ###
+	
+	implementation fromURL(
+		'https://github.com/kbase/jars/raw/master/lib/jars/kbase/common/kbase-common-0.2.0.jar',
+		'kbase-common-0.2.0'
+	)
+	// joda-time is required for kbase-common and syslog4j
+	implementation 'joda-time:joda-time:2.2'
+	// this is OOOOOOLD. But that probably means updating java_common
+	implementation 'org.eclipse.jetty.aggregate:jetty-all:7.0.0.v20091005'
+	implementation 'javax.servlet:servlet-api:2.5'
+	implementation 'javax.annotation:javax.annotation-api:1.3.2'
+	// Syslog4j 0.9.46 doesn't appear to be available on Maven. It apparently lives in 
+	// a JetBrains artifact server, but that's too much trouble and there's only one version there
+	// anyway.
+	// https://mvnrepository.com/artifact/org.jetbrains/syslog4j/0.9.46
+	// Need to rework the java common logger to not use syslog4j at all since it's abandonware
+	// and has a ton of CVEs, even in the newer versions.
```

Comment:
In the next commit I get rid of the JNA dependency since it prevents the server from working in a shadowjar, so I don't actually want to pull it from a repo since that'll force the JNA dependency. Plus it's no harder to pull it from jars. Plus plus we need to get rid of sylog4j anyway so a janky pulldown makes that more obvious. I'll update the comment to be more accurate along those lines